torch/csrc/api/include/torch/nn/options/loss.h

```diff
@@ -24,7 +24,7 @@ struct TORCH_API HingeEmbeddingLossOptions {
   /// reach in order to incur zero loss. Default: 1
   TORCH_ARG(double, margin) = 1.0;
   /// Specifies the reduction to apply to the output. Default: Mean
-  TORCH_ARG(Reduction::Reduction, reduction) = Reduction::Mean;
+  TORCH_ARG(at::Reduction::Reduction, reduction) = at::Reduction::Mean;
```

Comment:
Ditto in this file regarding changing `at::` to `torch::`.

Reply:
I may be totally wrong in my understanding. But I was amazed at how the code would work with `torch::` instead of `at::`. So, I tried to dig deeper and found the types.h class which wrap around the object with `torch` namespace and with `using namespace at` we are essentially removing the need of use of `at::`. Please do correct me if my understanding is wrong.

Reply:
Your understanding is 100% correct :D